internal/commands/exec/command.go

```diff
@@ -151,11 +151,6 @@ func (c *Command) Run(args []string) (ret int) {
 	if cfg.TLSConfig.CAFile != "" {
 		cfg.Scheme = "https"
 	}
-	consulClient, err := api.NewClient(cfg)
-	if err != nil {
-		logger.Error("error creating consul client", "error", err)
-		return 1
-	}
```

Comment:
The call to `api.NewClient` actually mutates the config under the hood -- for now let's just keep this and ignore the returned client, so:

```go
	// this call mutates the cfg object with a bunch of defaults
	// so we're going to keep it for now
	if _, err := api.NewClient(cfg); err != nil {
		logger.Error("error creating consul client", "error", err)
		return 1
	}
```

Reply:
Ugh that's an unfortunate behavior.